webpack.config.js

```diff
@@ -23,10 +20,20 @@ module.exports = {
 				loader: 'vue-loader'
 			},
 			{
-				test: /\.css$/,
+				test: /\.s?css$/,
 				use: [
 					'style-loader',
-					'css-loader'
+					'css-loader',
+					{
+						loader: 'sass-loader',
+						options: {
+							additionalData: `@import "/src/styles/globalStyles.scss";`,
+							sassOptions: {
+								data: '@import "/src/styles/globalStyles.scss";',
+								webpackSucks: "true"
```

Comment:
webpack doesn't suck, it's actually amazing and does exactly what you need it to do.

Reply:
Added it as a joke when talking with Romans and forgot to delete it haha, it turned out that the one who actually sucked was me😂